src/app/login/page.tsx

```diff
@@ -48,12 +51,33 @@ export default function App() {
           height={65}
           style={{ marginTop: '30px', marginLeft: '30px' }}
         />
+
         <LoginBox>
           <LoginContent>
             <WelcomeSign>Welcome</WelcomeSign>
```

Comment:
I know you didn't work on this but it would be great if you could please add the proper text component around welcome 🙏